### PR TITLE
Update HTML Proofer and use without bundler

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
@nschonni this makes it easier to integrate with Docker

File: .travis.yml [hunk 1]
```diff
@@ -8,9 +8,12 @@ env:
   global:
     - TRAVIS_COMMIT_MSG="$(git log --format=%B --no-merges -n 1)"
 
+before_install:
+  - rvm install 2.2.2
```
Comment (nschonni) [inline .travis.yml]: Not sure what this is for, I believe the machines already have RVM installed

File: script/docker2/update.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+cp Dockerfile.template Dockerfile
+
+echo "\n#\n# Java\n#\n" >> Dockerfile
+curl -sSL --compressed  https://raw.githubusercontent.com/docker-library/openjdk/9a0822673dffd3e5ba66f18a8547aa60faed6d08/8-jre/alpine/Dockerfile | tail -n +8  >> Dockerfile
+
+echo "\n#\n# Ruby\n#\n" >> Dockerfile
+#curl -sSL --compressed  https://raw.githubusercontent.com/docker-library/ruby/bfc7a48724ceb1917ddbcb713b24c835eca584c8/2.4/alpine/Dockerfile | tail -n +3 | head -n -1  >> Dockerfile
```
Comment (nschonni) [inline script/docker2/update.sh]: Not sure why this is included and commented out

File: script/docker2/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FROM node:6
```
Comment (nschonni) [inline script/docker2/Dockerfile]: These should be separate from this change

**Review by nschonni — DISMISSED** [review]
I think you can do your Bundler-less version for Docker if you want, but I don't think the Travis and core stuff should.
It was setup this way because this was one of the libraries that would regularly start breaking the builds if we left it as a wildcard dependency

File: script/docker2/update.sh (hunk 2 above)
Comment (LaurentGoderre) [inline script/docker2/update.sh]: These files should not have made it in...

File: .travis.yml (hunk 1 above)
Comment (LaurentGoderre) [inline .travis.yml]: The latest vesrion of HTML Proofer requires 2.2 but 1.9 is installed by default

**Comment by LaurentGoderre** [comment]
The goal is to transition to a Docker build for CI

**Comment by LaurentGoderre** [comment]
@nschonni can we try it for a while?

**Review by lucas-hay — APPROVED** [review]
Approved at PR review meeting.

**Pull request merged**